truffle.js

```diff
@@ -8,5 +8,10 @@ module.exports = {
   },
   mocha: {
     enableTimeouts: false
+  },
+  compilers: {
```

Comment:
What do you think about turning on compiler optimizations? Likely our last chance to do it.

Reply:
We should enable it , I think that is controlled by flags in the compilation before deployment. @nithinkrishna https://solidity.readthedocs.io/en/v0.4.24/using-the-compiler.html

Reply:
We should enable it here. 

In our deployment script, we run `truffle compile` in the context of this repository, so it uses the configuration in `market-oracle/truffle.js`.

https://github.com/ampleforth/uFragments-eth-integration/blob/master/deploy/helpers/compile-contracts.sh#L10

We should also update the truffle.js in the uFragments repository.